### PR TITLE
Add optional parameter stripe-account with test coverage

### PR DESCRIPTION
**Pull request by whiteancient**

### CODE REVIEW

File: src/StripeBase.ts [hunk 1]
```diff
@@ -388,13 +392,14 @@ export class StripeBase extends LitElement {
    * Initializes Stripe and elements.
    */
   private async initStripe(): Promise<void> {
-    const { publishableKey } = this;
+    const { publishableKey, stripeAccount } = this;
     if (!publishableKey)
       readonly.set<StripeBase>(this, { elements: null, element: null, stripe: null });
     else {
       try {
+        const options = stripeAccount ? { 'stripeAccount': stripeAccount } : {};
```
Comment (bennypowers) [inline src/StripeBase.ts:400]:
```suggestion
        const options = { stripeAccount };
```

File: test/mock-stripe/index.ts [hunk 2]
```diff
@@ -6,6 +6,7 @@ import { spy } from 'sinon';
 const assign = (target: object) => ([k, v]: [string, unknown]): unknown => target[k] = v;
 
 export enum Keys {
+  STRIPE_ACCOOUNT = 'acct_XXXXXXXXXXX',
```
Comment (bennypowers) [inline test/mock-stripe/index.ts:9]:
```suggestion
  STRIPE_ACCOUNT = 'acct_XXXXXXXXXXX',
```

File: test/stripe-elements.test.ts [hunk 3]
```diff
@@ -296,6 +296,17 @@ describe('<stripe-elements>', function() {
           });
         });
       });
+
+      describe('with a valid stripe account', function() {
+        beforeEach(Helpers.setupWithPublishableKeyAndStripeAccount(Keys.PUBLISHABLE_KEY, Keys.STRIPE_ACCOOUNT));
```
Comment (bennypowers) [inline test/stripe-elements.test.ts:301]:
```suggestion
        beforeEach(Helpers.setupWithPublishableKeyAndStripeAccount(Keys.PUBLISHABLE_KEY, Keys.STRIPE_ACCOUNT));
```

File: test/stripe-payment-request.test.ts [hunk 4]
```diff
@@ -455,6 +456,17 @@ describe('<stripe-payment-request>', function() {
           });
         });
 
+        describe('when stripe account is changed', function stripeAccountSet() {
+          beforeEach(Helpers.setProps({ publishableKey: 'foo', stripeAccount: 'bar' }));
+          beforeEach(nextFrame);
+          it('reinitializes stripe', function() { expect(element.stripe).to.be.ok.and.not.equal(Helpers.initialStripe); });
```
Comment (bennypowers) [inline test/stripe-payment-request.test.ts:462]:
```suggestion
          it('reinitializes stripe', function() { expect(element.stripe).to.be.ok.and.not.deep.equal(Helpers.initialStripe); });
```

File: docs/index.md [hunk 5]
```diff
@@ -46,6 +46,8 @@ Enter a valid _testing_ publishable key in the input below to activate the examp
 
 Once you've set the `publishable-key` attribute (or the `publishableKey` DOM property), Stripe will create a Stripe Card Element on your page.
 
+For use with Stripe Connect, you can provide the optional `stripe-account` attribute set to the account ID of the connected account.
```
Comment (bennypowers) [inline docs/index.md:49]:
```suggestion
For use with [Stripe Connect](https://stripe.com/docs/js/initializing#init_stripe_js-options-stripeAccount), you can provide the optional `stripe-account` attribute set to the account ID of the connected account.
```

**Review by bennypowers — CHANGES_REQUESTED** [review]
a few small nits

**Comment by whiteancient** [comment]
All your change requests committed :)

**Comment by bennypowers** [comment]
Thank you for the contribution

**Comment by whiteancient** [comment]
Thanks for the support to get it through!

**Comment by bennypowers** [comment]
This was released with 3.2.0

**Pull request merged**